docs/pages/router/reference/troubleshooting.mdx

```diff
@@ -5,6 +5,16 @@ description: Fixing common issues with Expo Router setup.
 
 import { Terminal } from '~/ui/components/Snippet';
 
+## Missing files / source maps in React Native DevTools
+
+This can happen if your Chrome DevTools has exclusions within its ignore list. To fix the issue
+
+1. Start the React Native DevTools
+2. Open settings by pressing the gear icon
```

Comment:
```suggestion
2. Open **Settings** by clicking the gear icon
```

In docs, all actions taken by a mouse reference to click instead of press (which is used for keyboard).